packages/components/src/SvgIcon/SvgIcon.tsx

```diff
@@ -89,15 +100,19 @@ function SvgIcon(props: SvgIconProps) {
     xmlns: "http://www.w3.org/2000/svg",
   };
 
-  if (props.role === "img") {
+  if (props.type === "informative") {
     return (
-      <svg {...svgCommonProps}>
+      <svg {...svgCommonProps} role="img">
         <title>{props.title}</title>
```

Comment:
I kinda wonder if we should use an `aria-label` and drop the role + title for informative icons for consistency. It feels a little weird to me that one version of the svgs has a role and the other doesn't. 🤷

Reply:
One benefit of `<title>` (which our auditors request sometimes) is it'll appear when hovering on the icon, in case it's not clear what an icon means. I do generally like aria-label better, but that could be a reason to keep this